api/src/geolocate/get.handler.ts

```diff
@@ -64,7 +87,7 @@ export const handler = async (event: {
   let body: GeolocateApiResponse = {};
 
   try {
-    await Promise.allSettled([
+    await Promise.all([
```

Comment:
Promise.allSettled is [never rejected](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled), so was swallowing errors.

Reply:
Great catch, thank you!

Reply:
:bowtie: